cnd/src/http_api/route_factory.rs

```diff
@@ -20,35 +16,35 @@ pub fn new_action_link(id: &SwapId, action: &str) -> String {
     format!("{}/{}", swap_path(*id), action)
 }
 
-pub fn create<D: MetadataStore + StateStore + AliceSpawner + BobSpawner + Network + Clone>(
+pub fn create<C: Connect>(
     origin_auth: String,
     peer_id: PeerId,
-    dependencies: D,
+    con: C,
```

Comment:
```suggestion
    connector: C,
```
Do you really need shortening?

Reply:
I want to second this. This kind of shortening only adds confusion without an immediate benefit (if you are a lazy typer, please configure your environment to give you autocompletion).

Reply:
Am renaming this in the next planned block of work, can we postpone this debate until then (I'll lean towards a full length name though).